### DISCUSSION TITLE
Improve test runs

### DISCUSSION THREAD

**Discussion by Erikborgis** [Erikborgis/react-automatic-testing#16] [General]
We need to somehow improve how the implementation of running tests works...

We need to figure out:

- [ ] How to make the test be able to import the files that the test was generated from???
- [x] See if the test running script works on an actually working test.

**Comment by Erikborgis** [comment]
So... how do we always make sure that the test file imports the right file?? We most likely need to have them in the same folder and always make sure that the file name uses the same classname??? or how will we make sure that this always works? 
If we dont manage to solve this it will generate a lot of failing tests....

**Comment by RunningCircles42** [comment]
This issue is fixed. I include the path to the react component in the prompt.
